mem-09-n.xml

```diff
@@ -880,7 +880,7 @@
       <column name="definition_fi">hedelmä, vihannes</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
-      <column name="see_also">{baQ:v:2}, {DeH:v}, {Du' naH:n}, {naH tlhab:n}, {ghub:n}, {qurgh:n}, {'oQqar:n}, {'aquta':n}, {yub:n}, {tIr:n}, {yob:v}, {wIj:v}, {yotlh:n}</column>
+      <column name="see_also">{baQ:v:2}, {DeH:v}, {Du' naH:n}, {naH tlhab:n}, {ghub:n}, {qurgh:n}, {'oQqar:n}, {'aquta':n}, {yub:n}, {tIr:n}, {yob:v}, {wIj:v}, {yotlh:n}, {peb'ot:n}</column>
```

Comment:
(This is probably a case where I should split off "examples of the thing" into a list under `notes`, as it's getting a bit long. Just noting to myself to do this later.)